src/main/java/it/gov/pagopa/payhub/ionotification/dto/mapper/IONotificationMapper.java

```diff
@@ -17,4 +22,20 @@ public NotificationDTO mapToQueue(String fiscalCode, Long timeToLive, String sub
                 .fiscalCode(fiscalCode)
                 .build();
     }
+
+    public FiscalCodeDTO mapToGetProfile(NotificationQueueDTO notificationQueueDTO) {
+        return FiscalCodeDTO.builder()
+                .fiscalCode(notificationQueueDTO.getFiscalCode())
+                .build();
+    }
+
+    public IONotification mapToSaveNotification(NotificationQueueDTO notificationQueueDTO, String status){
+        return IONotification.builder()
+                .notificationDate(LocalDateTime.now())
+                .userId(notificationQueueDTO.getFiscalCode())
+                .notificationStatus(status)
```

Comment:
may it be appropriate to use the notification status enum in the models? if you think the refactor is too much extended, maybe you can move it to a new PR. But overall codify the status as a String is weak and not type safe